src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -121,17 +118,17 @@ export const getModuleDetails = async (
   );
 
   // TODO: Customize this block lookback based on chain and test with L2 network (Polygon)
-  const events = await oracleContract.queryFilter('ProposePrice', -100000);
-  const proposeEvent = events.filter(
+  const proposalEvents = await oracleContract.queryFilter('ProposePrice');
+  const thisModuleProposalEvents = proposalEvents.filter(
     event =>
       event.args?.requester === moduleAddress &&
       event.args?.ancillaryData === ancillaryData &&
       Number(event.args?.timestamp) === Number(timestamp)
   );
```

Comment:
@pemulis here's how you do it. See [the event](https://github.com/UMAprotocol/protocol/blob/5726cd16cd74b96aab82d6f99c9683e3d72b6723/packages/core/contracts/optimistic-oracle/interfaces/OptimisticOracleV2Interface.sol#L21-L30) for reference.


```suggestion
  const proposalEvents = await oracleContract.queryFilter(oracleContract.filters.ProposePrice(moduleAddress));
  const thisModuleProposalEvents = proposalEvents.filter(
    event =>
      event.args?.ancillaryData === ancillaryData &&
      Number(event.args?.timestamp) === Number(timestamp)
  );
```